pom.xml

```diff
@@ -261,36 +259,6 @@ com.fasterxml.jackson.core.*;version=${project.version}
           </archive>
         </configuration>
       </plugin>
-
-      <!-- 23-Mar-2023, tatu: [core#965] Need to put back Gradle module metadata marker -->
-      <plugin>
-        <groupId>io.github.floverfelt</groupId>
-        <artifactId>find-and-replace-maven-plugin</artifactId>
-        <version>1.1.0</version>
-        <executions>
-          <execution>
-            <id>exec</id>
-            <phase>package</phase>
-            <goals>
-              <goal>find-and-replace</goal>
-            </goals>
-            <configuration>
-              <replacementType>file-contents</replacementType>
-              <baseDir>${basedir}</baseDir>
-              <findRegex><![CDATA[<modelVersion>4.0.0</modelVersion>]]></findRegex>
-              <fileMask>dependency-reduced-pom.xml</fileMask>
-              <replaceValue><![CDATA[  <!-- This module was also published with a richer model, Gradle metadata,  -->
```

Comment:
The `gradle-module-metadata-maven-plugin` is now doing this "replacement" directly if the Marker Comment is missing.